courses/machine_learning/deepdive2/text_classification/solutions/classify_text_with_bert.ipynb

```diff
@@ -0,0 +1,1392 @@
+{
```

Comment:
<p>activation=none defaults to linear right? Since you apply the sigmoid below to normalize the positive sentiment class probability, what do you think about moving up to condense? Or making that explicit activation=linear for students to make that connection</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/1412/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

Reply:
<p>That's right, it would be a linear layer as the last layer. I moved the code cell with Sigmoid up and then added activation=linear as you suggested since that is what they have seen through ASL.</p>